projects/addon-mobile/components/mobile-calendar/mobile-calendar.template.html

```diff
@@ -73,13 +73,16 @@
         *cdkVirtualFor="let month of months; templateCacheSize: 10"
         class="t-month-wrapper"
     >
-        <h2 class="t-month">{{ month | tuiMonth | async }}</h2>
-        <tui-primitive-calendar-mobile
-            class="t-calendar"
-            [disabledItemHandler]="disabledItemHandler | tuiMapper: disabledItemHandlerMapper : min : max"
-            [month]="month"
-            [value]="value"
-            (dayClick)="onDayClick($event)"
-        ></tui-primitive-calendar-mobile>
+        <ng-container *ngIf="month | tuiMonth | async as monthTitle">
+            <h2 class="t-month">{{ monthTitle }}</h2>
+            <tui-primitive-calendar-mobile
+                class="t-calendar"
+                [attr.data-month]="monthTitle | lowercase"
```

Comment:
Is this used anywhere?

Reply:
in e2e test

Reply:
Let's not do that, it doesn't make much sense — adding a non-specific attribute binding just for the test

Reply:
fixed